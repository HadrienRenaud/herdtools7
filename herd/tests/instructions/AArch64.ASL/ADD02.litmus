AArch64 ADD02
{
0:X0=1;
0:X1=-2;
int64_t 0:X2;
}
  P0                ;
 ADD W2,W1,W0       ;
forall 0:X2=4294967295