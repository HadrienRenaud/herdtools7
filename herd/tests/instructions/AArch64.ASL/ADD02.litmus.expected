Test ADD02 Required
States 1
0:X2=4294967295;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X2=4294967295)
Observation ADD02 Always 1 0
Hash=502f00a12d9a45353343894fb2a98845

